GameData/RP-0/Contracts/Satellite/WeatherContracts.cfg

```diff
@@ -437,18 +437,18 @@ CONTRACT_TYPE
 			name = NewVessel
 			type = NewVessel
 		}
-                PARAMETER
```

Comment:
While this looks the same in the github viewer, I replaced spaces with tabs, since the rest of the file uses tabs.